sources/model/entity/Calendar.ts

```diff
@@ -0,0 +1,94 @@
+import { Entity, PrimaryGeneratedColumn, ManyToOne, Column, ManyToMany, OneToMany, Repository } from "typeorm";
+import { Event } from "./Event"
+import { User } from "./User";
+
+@Entity("calendars")
+export class Calendar {
+
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({length: 256})
+    name: string
+
+    @OneToMany(type => Event, event => event.calendar)
+    events: Event[]
+
+    @ManyToMany(type => User, user => user.calendars)
+    owners: User[]
```

Comment:
A `Calendar` can belong to several `User` ?

I was imagining something more like:

Group 1 => * Calendar
User 1 => * Calendar